json_rpc/clients/httpclient.nim

```diff
@@ -84,6 +87,7 @@ method call*(client: RpcHttpClient, name: string,
         await res.closeWait()
       except CatchableError as exc: # shouldn't happen
         debug "Error closing JSON-RPC HTTP resuest/response", err = exc.msg
+    {.hint[XDeclaredButNotUsed]:on.}
```

Comment:
this turns it on even if it was disabled before.. why not use `{.used.}`?

Reply:
fixed/removed